spec/lib/jobs/obm-control-spec.js

```diff
@@ -131,6 +132,63 @@ describe("Job.Obm.Node", function () {
             });
         });
 
+        it('should fail if there is more then one OBM service', function(done){
+          var node = {
+              obmSettings: [
+                  {
+                      service: 'noop-obm-service',
+                      config: {}
+                  },
+                  {
+                      service: 'foo',
+                      config: {
+                          "user": "admin",
+                          "password": "admin",
+                          "host": "10.0.0.254"
+                      }
+                  }
+              ]
+          };
+          mockWaterline.nodes.findByIdentifier.resolves(node);
+
+          job.run()
+          .then(function() {
+              done(new Error("Expected job to fail"));
+          })
+          .catch(function(e) {
+              try {
+                  expect(e).to.have.property('name').that.equals('AssertionError');
+                  expect(e).to.have.property('message').that.equals(
+                      'more than one OBM service assigned to this node,' +
+                      ' no default set for which to use'
+                  );
+                  done();
+              } catch (e) {
+                  done(e);
+              }
+          });
+        });
```

Comment:
ok, I got help from @VulpesArtificem to wrap my head around what I was doing wrong... and I updated the code to implement the changes you requested, fix will be in the next push.